.github/workflows/test-cli-with-database.yml

```diff
@@ -59,7 +59,7 @@ jobs:
 
       - name: Create Medusa project
         run: |
-          medusa new cli-test
+          medusa new cli-test 'https://github.com/medusajs/medusa-starter-default/tree/typeorm-upgrade'
```

Comment:
self-review: This will be changed later, at the moment it depends on it to match the typeorm expected version from the core in this pr

Reply:
added a todo